### ISSUE TITLE
Use SVGs for all icons

### ISSUE THREAD

**Issue by jenniferthibault** [18F/doi-extractives-data#2821]
From @brentryanjohnson in #2697: 

> A few months ago, we replaced some font icons with SVGs, but not all. It was determined SVG icons are a better long-term solution, in large part because they are superior for accessibility. We'll need to update the style guide to reflect these changes as we go. And, as indicated in #2696, we are still missing some icons altogether.

![icon audit comparing svg format with font icons](https://user-images.githubusercontent.com/32855580/38589239-bd5f4912-3cde-11e8-98b5-0e12d28ca4b7.png)

- [ ] Replace non-SVG icons with SVGs
- [ ] Update the site instances
- [ ] Update style guide

**Comment by jennmalcolm** [comment]
closing in favor of issue above
